include/simple/server.hpp

```diff
@@ -53,7 +53,8 @@ class Server {
    * @param [in] linger - Time the unsent messages linger in memory after the socket
```

Comment:
There's some copy-past mistakes in the comments of this file (which I can't mark directly because they are not changed) which mention "Subscriber" instead of "Server". I saw some in the move assignment operator and in the comment for `isValid()`, there may be more